autoload/lsp/omni.vim

```diff
@@ -288,6 +295,11 @@ function! lsp#omni#get_vim_completion_items(options) abort
         let l:incomplete = 0
     endif
 
+    if len(l:items) > 0 && has_key(l:items[0], 'sortText')
+      " If first item contains sortText, maybe we can use sortText
+      call sort(l:items, function('s:sort_by_sorttext'))
+    endif
```

Comment:
This is my opinion. DO NOT CHANGE CODES AGAINST THIS REVIEW COMMENT PLEASE.

Document says:
> A string that should be used when comparing this item with other items. When falsy the label is used as the sort text for this item.

So the sorting should always be done always like.

```vim
function! s:sort(i1, i2) abort
  let l:text1 = get(a:1, 'sortText', get(a:1, 'label', ''))
  let l:text2 = get(a:2, 'sortText', get(a:2, 'label', ''))
  return l:text1 == l:text2 ? 0 : l:text1 > l:text2 ? 1 : -1
endfunction
```

Reply:
Thank you for review comment.

I'm worried about breaking existing behavior and performance.
So I intent to sorting only LS supported

@prabirshrestha mention about performance impact I agree that.

what about do someting like this?
User can config sort by each LS server but LS does not support sort, sort by label.

```
" per-server configuration options

config: { 'sort': { 'max': 1000 } }
```

```
function! s:sort(i1, i2) abort
  let l:text1 = get(a:1, 'sortText', get(a:1, 'label', ''))
  let l:text2 = get(a:2, 'sortText', get(a:2, 'label', ''))
  return l:text1 == l:text2 ? 0 : l:text1 > l:text2 ? 1 : -1
endfunction

function! lsp#omni#get_vim_completion_items(options) abort
  " snip

  let l:sort = has_key(l:server_info, 'config') && has_key(l:server_info['config'], 'sort') ? l:server_info['config']['sort'] : { 'max': '1000' }

  if len(l:items) > l:sort['max']
    call sort(l:items, s:sort)
  endif

  " snip
endfunction
```

Reply:
`label` property is not optional.

```vim
function! s:sort(i1, i2) abort
  let l:text1 = get(a:1, 'sortText', a:1['label'])
  let l:text2 = get(a:2, 'sortText', a:2['label'])
  return l:text1 == l:text2 ? 0 : l:text1 > l:text2 ? 1 : -1
endfunction
```

This would slightly be better in terms of performance.

Reply:
And one more point. Falsy values include empty string. When `sortText` property is empty string, `label` should be used instead.

Reply:
Thank you for comment!

I just update when sortText is empty string, use label instead and get label directory(`a:1['label'])`)